tutorials/vue/vue-select.md

```diff
@@ -0,0 +1,104 @@
+To create a dropdown in vanilla javascript, you must use the `<select>`
+tag in your template option. Vue makes this much easier to do with its
+list rendering functionality. To begin, you must use `v-model` with
+your select tag to to inform the data what value it should be storing
+from the options presented to the user. Once that is done, you may type
+the options and vue will handle the rest, as shown below:
```

Comment:
Replace paragraph with "In HTML, the `<select>` tag lets you create a dropdown that lets the user select one of several options. The easiest way to tie Vue state to the value of a `<select>` is using `v-model`. Below is an example."